crates/nostr/src/types/metadata.rs

```diff
@@ -238,8 +238,9 @@ where
             let mut custom_fields: AllocMap<String, String> =
                 AllocMap::with_capacity(map.size_hint().unwrap_or_default());
             while let Some(field_name) = map.next_key::<String>()? {
-                let value: String = map.next_value()?;
-                custom_fields.insert(field_name, value);
+                if let Ok(value) = map.next_value::<String>() {
```

Comment:
@benthecarman is this the right approach? I would imagine it needs to just take in a `::<Value>`?

Reply:
Changed to be value